index.html

```diff
@@ -14,6 +14,373 @@
   <title>Itumeleng Rantsane | Portfolio </title>
 </head>
 
+<<<<<<< HEAD
+  <body>
+    <div class="hero">
+      <header class="header bg-white">
+        <nav class="navbar flex">
+          <a href="#" class="navbar__link link fz-20 fw-700 lh-20">I.R</a>
+          <div class="navbar__menu">
+            <ul class="navbar__menu__ul ul flex">
+              <li class="navbar__menu__li">
+                <a
+                  href="#works"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Portfolio</a
+                >
+              </li>
+              <li class="navbar__menu__li">
+                <a
+                  href="#about"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >About</a
+                >
+              </li>
+              <li class="navbar__menu__li ">
+                <a
+                  href="#contact"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Contact</a
+                >
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] Please apply the bluer effect on the background to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Chuai4vEE9yvqNFx-0) design.
  - For filter, you can use `filter: blur(Npx);`. N is an integer number for blur capacity.

### Your design
![image](https://user-images.githubusercontent.com/83803414/203592148-a2ed92fc-9cc6-49c7-bc27-54931882f6d3.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/203592189-184a439f-a7aa-40e8-843c-768b42daa1e1.png)